tests/database/odbc-with-cksp/enclave-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "args": [
+        "odbc_app",
+        "classic",
+        "default"
+    ],
+    "host_import_env": [
+        "MAA_URL",
```

Comment:
Where is MAA_URL used?

Reply:
In `odbc_app.c` line:47. It is used as part of the SQL Server Always Encrypted connection string